gnat2goto/driver/tree_walk.adb

```diff
@@ -3887,6 +3901,13 @@ package body Tree_Walk is
          Func_Symbol.Mode := Intern ("C");
          Func_Symbol.Value := Body_Block;
          Global_Symbol_Table.Insert (Intern (Func_Name), Func_Symbol);
+         --  Add allocation function to symbol table
+         Alloc_Symbol.SymType := Func_Type;
+         Alloc_Symbol.Name := Intern (Alloc_Name);
+         Alloc_Symbol.PrettyName := Alloc_Symbol.Name;
+         Alloc_Symbol.BaseName := Alloc_Symbol.Name;
+         Alloc_Symbol.Mode := Intern ("C");
```

Comment:
I don't see any use of Alloc_Symbol.Value (so no body?) - or further use of Alloc_Symbol. 
Just flagging this. 
Not enough gnat2goto experience to say if this is likely a problem going forward.

Reply:
Here CBMC was complaining that `__new__array` is not in the symbol table, but it only needed declaration: the definition is in the builtin functions. So I thought the body would be ignored anyway.

Reply:
Empty body might do, but I'm not sure how to create one in gnat2goto. `Make_Nil` results in the same warning and `New_Irep (I_Code_Block)` breaks CBMC.

Reply:
Ok. Thanks for extra effort + info - will look into it as part of the array epic.